README.rst

```diff
@@ -49,18 +49,20 @@ template files are included in the report.
 Caveats
 ~~~~~~~
 
-Files included by the ``{% ssi %}`` tag are not included in the coverage
-measurements.
-
 Coverage.py can't tell whether a ``{% blocktrans %}`` tag used the singular or
 plural text, so both are marked as used if the tag is used.
 
 
 Changes
 ~~~~~~~
 
+v1.5.1b --- 2017-10-18
```

Comment:
Why this version number? Shouldn't it be 1.5.2? Or 1.5.1b2?

Reply:
I was mirroring the previous version number of `v1.5.1a` and thinking of waiting for Django 2 to be released, but I'll just call it 1.5.2 so the world is simple and go from there.